src/containers/my-quizzes/QuizzesContainer.tsx

```diff
@@ -46,20 +62,49 @@ const TestsContainer = () => {
     [itemsPerPage]
   )
 
-  const { response, loading } = useAxios<ItemsWithCount<Quiz>>({
+  const deleteQuiz = useCallback(
+    (id?: string) => quizService.deleteQuiz(id ?? ''),
```

Comment:
I think that question mark is useless here. You need to use this `(id: string) => quizService.deleteQuiz(id),`
Why sent a request to delete the quiz with an empty string?